.github/dependabot.yaml

```diff
@@ -58,39 +122,3 @@ updates:
           - "playwright"
           - "black"
           - "djlint"
-    ignore:
-      # Temporarily pinned to v2.14.0 - problem for translations found in v2.15.0, see: https://github.com/ONSdigital/eq-questionnaire-runner/pull/1384
```

Comment:
We might want to keep this in (hopefully only temporary) as the minor version bump for babel affects the tests?